### PR TITLE
Add idh dev cluster

### PR DESCRIPTION
**Pull request by anishasthana**

### CODE REVIEW

File: manifests/overlays/prod/secrets/clusters/secret-generator.yaml [hunk 1]
```diff
@@ -4,5 +4,7 @@ metadata:
   name: cluster-spec-generator
 files:
 - secrets/clusters/apps.ocp.prod.psi.redhat.com.enc.yaml
+- secrets/clusters/api.aasthana.dev.datahub.redhat.com.enc.yaml
```
Comment (accorvin) [inline manifests/overlays/prod/secrets/clusters/secret-generator.yaml:7]: Was this change supposed to be part of this PR?

Comment (anishasthana) [reply manifests/overlays/prod/secrets/clusters/secret-generator.yaml:7]: yep!

Comment (anishasthana) [reply manifests/overlays/prod/secrets/clusters/secret-generator.yaml:7]: (we forgot to add this line in a previous PR)

File: manifests/overlays/prod/secrets/clusters/api.ocp4.idh.dev.datahub.redhat.com.enc.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        tlsClientConfig:
```
Comment (HumairAK) [inline manifests/overlays/prod/secrets/clusters/api.ocp4.idh.dev.datahub.redhat.com.enc.yaml:14]: looks like you have the jwt token as yaml, it should be json, i.e. of the form: 
```
config: | 
     {"bearerToken": "token", "tlsClientConfig": {"insecure: true}} 
```
what you have: 
```
config: 
   bearerToken: token
   tlsClientConfig: 
      insecure: true
```

Comment (anishasthana) [reply manifests/overlays/prod/secrets/clusters/api.ocp4.idh.dev.datahub.redhat.com.enc.yaml:14]: fixed

**Pull request merged**